config/dataset/test_ci_segmentation_binary.yaml

```diff
@@ -15,4 +15,4 @@ dataset:
   class_name: # will follow in the next version
   classes_dict: {'BUIL':1}
   class_weights:
-  ignore_index: 255
+  ignore_index: -1
```

Comment:
why ignore_index = -1? it is  a convention to use 255 for 8-bit data.

Reply:
yeah but it was like that before so I rechange it to the original value

Reply:
I think that 255 is the correct way to mark ignor_index pixels.
Let's hear what other mates think